tests/test_interpreter.py

```diff
@@ -15,6 +18,17 @@
 from pipx.util import PipxError
 
 
+@pytest.fixture()
+def mocked_github_api(monkeypatch, root):
+    """
+    Fixture to replace the github index with a local copy,
+    to prevent unit tests from exceeding github's API request limit.
+    """
+    with open(root / "testdata" / "standalone_python_index.json") as f:
+        index = json.load(f)
+    monkeypatch.setattr(pipx.standalone_python, "get_or_update_index", lambda: index)
```

Comment:
Actually, patching the `index_file` variable here would enable us dropping the `json` import?

```suggestion
    index_file = root / "testdata" / "standalone_python_index.json"
    monkeypatch.setattr(pipx.standalone_python, "get_or_update_index", lambda: index_file)
```

Reply:
the return value of `get_or_update_index` is a dict, not a file

Reply:
Oh, I wasn't aware that one can't patch local variables!